test/features/support/oidc_introspect.js

```diff
@@ -0,0 +1,132 @@
+const { spec, stash } = require("pactum");
+const { Given, When, Then, Before, After } = require("@cucumber/cucumber");
+const { localhost } = require("./helpers/helpers");
+
+const baseUrl = `${localhost}introspect`;
+
+let specIntrospect;
+
+stash.addDataTemplate({
+  QueryParams: {
+    client_id: "string",
+    token: "string.string.string",
+    token_type_hint: "access_token",
+  },
+});
+
+stash.addDataTemplate({
+  ResponseSchemaOK: {
+    type: "object",
+    properties: {
+      active: { type: "boolean" },
+    },
+    required: ["active"],
+    additionalProperties: true,
+  },
+});
+
+stash.addDataTemplate({
+  ResponseSchemaUnauthorized: {
+    type: "object",
+    properties: {
+      error: { type: "string" },
+      error_description: { type: "string" },
+    },
+    required: ["error", "error_description"],
+    additionalProperties: false,
+  },
+});
+
+const resultFunction = async (
```

Comment:
```suggestion
const resultValidator = async (
```
I made the same mistake. 'resultFunction' does not say much and here we want to validate response, right?